source/requirements.rst

```diff
@@ -2,7 +2,7 @@ Requirements
 ============
 Before installing Quipucords in your environment, review the following guidelines about installing and running Quipucords:
 
-- You must install Quipucords on a system that is running Red Hat Enterprise Linux 6 or 7, CentOS 7, or Fedora 27 or 28. For an installation on Fedora, the scripted installation of Quipucords might not work correctly on all systems. Therefore, the best practice for a Fedora installation is to use the step-by-step installation instructions.
+- You must install Quipucords on a system that is running Red Hat Enterprise Linux 6/7 or CentOS 6/7.
```

Comment:
Linux 6 or 7, not Linux 6/7 (and Centos 6 or 7)
Reasons this 6/7 is bad:
- Linux 6/7 is *not* the name of a Linux release.
- Don't use symbols to substitute for words in documentation. 
- This would not be considered to meet accessibility standards for good text for a screen reader.

Reply:
@jlprevatt So really we have 4 options (no and).

How about this?
```
You must install Quipucords on a system that is running Red Hat Enterprise Linux 6 or 7 or CentOS 6 or 7.
```

Reply:
sorry if my parentheses caused confusion about how to word the change...
yes, that's fine

Reply:
Fixed.